examples/pytorch/semantic-segmentation/requirements.txt

```
@@ -0,0 +1,3 @@
+accelerate
```

```suggestion
git://github.com/huggingface/accelerate.git
```
Hoping to have a release next week when Sourab as finished integrating FSDP, so we can pin this one instead, but in the meantime an install from master is better.